modules/weko-workflow/weko_workflow/views.py

```diff
@@ -727,7 +728,8 @@ def cancel_action(activity_id='0', action_id=0):
                 cancel_deposit.clear()
                 # Remove draft child
                 cancel_pid = PersistentIdentifier.get_by_object(
-                    pid_type='recid', object_type='rec', object_uuid=cancel_item_id)
+                    pid_type='recid', object_type='rec',
```

Comment:
Modify source code/ files not in the ticket scope

Reply:
Thank for your comment.
It had been fix at last rev.